piff/optical_model.py

```diff
@@ -175,52 +268,211 @@ def fit(self, star):
 
         # compute chisq
         chisq = np.std(image.array - model_image.array)
-        dof = np.count_nonzero(weight.array)
+        dof = np.count_nonzero(weight.array) - 6
```

Comment:
Where does 6 come from?  Probably deserves an inline comment.

Reply:
Thinking about this now, can only identify 3 dof from fitting - for flux and centroid.  Changed to 3 and commented